Java/Stack/Stack.java

```diff
@@ -0,0 +1,77 @@
+public class Stack {
```

Comment:
Hey @Khushi2402 

I've merged the PR.
But at the start of every program, we need to write a comment about the program and what problem it is solving. It really helps to know about the program. 

In this case, we can write:
`Program to implement Stack using Java`